src/test/scala/chiselTests/util/PlusArgsValueSpec.scala

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests.util
+
+import chisel3._
+import chisel3.testers.BasicTester
+import chisel3.util.circt.PlusArgsValue
+import circt.stage.ChiselStage
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.io.Source
+
+private class PlusArgsValueTop extends Module {
+  val wf = IO(Output(UInt(1.W)))
+  val wv = IO(Output(UInt(32.W)))
+  val xf = IO(Output(UInt(1.W)))
+  val xv = IO(Output(UInt(32.W)))
+  val zv = IO(Output(UInt(32.W)))
+  val tmpw = PlusArgsValue(UInt(32.W), "FOO=%d")
+  val tmpx = PlusArgsValue(xv, "BAR=%d")
+  zv := PlusArgsValue(xv, "BAR=%d", 42.U)
+  wf := tmpw.found
+  wv := tmpw.result
+  xf := tmpx.found
+  xv := tmpx.result
+}
+
+class PlusArgsValueSpec extends AnyFlatSpec with Matchers {
+  it should "Should work for types" in {
+    val fir = ChiselStage.emitCHIRRTL(new PlusArgsValueTop)
+    println(fir)
+    (fir.split('\n').map(_.trim) should contain).inOrder(
```

Comment:
The PR can do a `takeWhile(_ != '@')` to strip the source locators before the trim. (Generally it's bad to match on the source locator.)